novawallet/Modules/DApp/DAppBrowser/DAppBrowserScriptHandler.swift

```diff
@@ -21,10 +21,10 @@ final class DAppBrowserScriptHandler: NSObject {
     }
 
     func bind(viewModel: DAppTransportModel) {
-        clearScript()
-
         self.viewModel = viewModel
 
+        clearScript(for: viewModel)
```

Comment:
Why we moved to the logic when we are cleaning the script for the new viewModel not for the old one?